src/exampleTypes/ResultTable/Header.js

```diff
@@ -80,10 +67,11 @@ let Header = ({
     hideMenu,
     typeDefault,
   } = fieldSchema
-  let HeaderCell = fieldSchema.HeaderCell || HeaderCellDefault
+  let HeaderCell = fieldSchema.HeaderCell || Th
   let filterNode =
     criteria &&
     _.find({ field }, _.getOr([], 'children', tree.getNode(criteria)))
+  let filtering = React.useState(!!_.get('hasValue', filterNode))
```

Comment:
This is a good idea, this will also make header filters be expanded by default if they are expanded in FilterList. Do you confirm we want that behavior?

Reply:
yeah, let's do it